charts/redpanda/post_install_upgrade_job.go

```diff
@@ -97,18 +97,18 @@ func PostInstallUpgradeJob(dot *helmette.Dot) *batchv1.Job {
 			Name:      fmt.Sprintf("%s-configuration", Fullname(dot)),
 			Namespace: dot.Release.Namespace,
 			Labels: helmette.Merge(
-				FullLabels(dot),
 				helmette.Default(map[string]string{}, values.PostInstallJob.Labels),
+				FullLabels(dot),
 			),
 			Annotations: helmette.Merge(
+				helmette.Default(map[string]string{}, values.PostInstallJob.Annotations),
```

Comment:
I'm guessing that we have this change since we don't want others to be able to override the helm-specific annotations?

Is there a perceived use-case where we want users to be able to override these?

Reply:
> I'm guessing that we have this change since we don't want others to be able to override the helm-specific annotations?

It's other way around. If someone would like to override (not recommended) with this change they will be able to change annotation. Merge from mergo works other way around. 

> Is there a perceived use-case where we want users to be able to override these?

As in the https://redpandacommunity.slack.com/archives/C01AJDUT88N/p1732833410021669 community member would like to delete Job after they succeed.